src/Slot-Tests/SlotTest.class.st

```diff
@@ -97,3 +97,8 @@ SlotTest >> testisUsed [
 	self assert: IndexedSlot isUsed. "subclasses are users"
 	self assert: ProcessLocalSlot isUsed. "references count as uses, too"
 ]
+
+{ #category : #'tests - misc' }
+SlotTest >> testslotUsers [
+	self assert: ToOneRelationSlot slotUsers first equals: SlotExampleMovie
```

Comment:
```suggestion
	self assert: (ToOneRelationSlot slotUsers includes: SlotExampleMovie).
	self deny: (ToOneRelationSlot slotUsers includes: SlotExamplePerson)
```

Reply:
I will do another PR with the improved test